### PR TITLE
Provide Microsoft.Extentions.FileProviders.Physical package readme

### PR DESCRIPTION
**Pull request by eNeRGy164**
Fill out details for the `Microsoft.Extentions.FileProviders.Physical` NuGet package readme.

See #92228

### CODE REVIEW

**Comment by ViktorHofer** [comment]
@dotnet/area-extensions-filesystem please review this package readme PR before EOW so that we can get the change still ported into .NET 8.

File: src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md [hunk 1]
```diff
@@ -2,43 +2,69 @@
...
 <!-- The main types provided in this library -->
 
 The main types provided by this library are:
 
-* ``
-* ``
-* ``
+* `PhysicalFileProvider`
+* `PhysicalDirectoryInfo `
+* `PhysicalFileInfo `
```
Comment (Jozkee) [inline src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md:50]:
```suggestion
* `Microsoft.Extensions.FileProviders.PhysicalFileProvider`
* `Microsoft.Extensions.FileProviders.Physical.PhysicalDirectoryInfo`
* `Microsoft.Extensions.FileProviders.Physical.PhysicalFileInfo`
```

File: src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md [hunk 2]
```diff
@@ -2,43 +2,69 @@
...
 
 ## Additional Documentation
 
 <!-- Links to further documentation. Remove conceptual documentation if not available for the library. -->
 
-* [Conceptual documentation](https://learn.microsoft.com/en-us/dotnet/standard/serialization/**LIBRARYNAME**/overview)
-* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/**LIBRARYNAME**)
+* [Conceptual documentation](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/file-providers#physical-file-provider)
+* [API documentation](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.fileproviders.physical)
```
Comment (Jozkee) [inline src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md:57]:
```suggestion
* [Conceptual documentation](https://learn.microsoft.com/aspnet/core/fundamentals/file-providers#physical-file-provider)
* [API documentation](https://learn.microsoft.com/dotnet/api/microsoft.extensions.fileproviders.physical)
```

File: src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md [hunk 3]
```diff
@@ -2,43 +2,69 @@
...
 
 ## Key Features
 
 <!-- The key features of this package -->
 
-*
-*
-*
+* Easy access and monitoring of files on the disk.
+* Capability to watch for file changes using `FileSystemWatcher` or polling.
```
Comment (danmoseley) [inline src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md:13]:
```suggestion
* Ability to watch for file changes either by using `FileSystemWatcher` or through polling.
```

File: src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md [hunk 4]
```diff
@@ -2,43 +2,69 @@
...
 <!-- The main types provided in this library -->
 
 The main types provided by this library are:
 
-* ``
-* ``
-* ``
+* `Microsoft.Extensions.FileProviders.PhysicalFileProvider`
+* `Microsoft.Extensions.FileProviders.PhysicalDirectoryInfo `
+* `Microsoft.Extensions.FileProviders.PhysicalFileInfo `
```
Comment (Jozkee) [inline src/libraries/Microsoft.Extensions.FileProviders.Physical/src/PACKAGE.md:50]: Whitespace
```suggestion
* `Microsoft.Extensions.FileProviders.PhysicalDirectoryInfo`
* `Microsoft.Extensions.FileProviders.PhysicalFileInfo`
```

**Review by jozkee — APPROVED** [review]
Thanks!

**Pull request merged**